multisensor_pipeline/tests/test_profiling.py

```diff
@@ -18,7 +18,6 @@ def test_frequency_stats(self):
         for frame in frames:
             msp_stats.add_frame(frame, direction=MSPModuleStats.Direction.IN)
             sleep(1. / frequency)
-        sleep(1)  # To make this work under macOS in the cloud
```

Comment:
@ombh0 Sleeping does not help in this case. Do you have a better idea or can you reproduce the test failures locally?